### PR TITLE
Use BrainIO

### PR DESCRIPTION
**Pull request by jjpr-mit**
Use BrainIO as a library with a local `lookup.csv`.

- [ ] before merging: pull the latest `lookup.csv` to make sure we're not leaving any PRs behind

### CODE REVIEW

File: test_setup.sh [hunk 1]
```diff
@@ -8,5 +8,5 @@ do
   aws --no-sign-request s3 cp s3://brain-score-tests/tests/test_benchmarks/${f} ${SCRIPT_DIR}/tests/test_benchmarks/
 done
 
-pip install git+https://github.com/brain-score/model-tools --no-deps
+pip install git+https://github.com/brain-score/model-tools@core_refactor --no-deps
```
Comment (mschrimpf) [inline test_setup.sh:11]: once this qualification is removed, I think this looks good

Comment (jjpr-mit) [reply test_setup.sh:11]: Of course, it’ll fail tests after that.

Comment (mschrimpf) [reply test_setup.sh:11]: so in what order do you want to merge? 1. brain-score, 2. model-tools?

Comment (jjpr) [reply test_setup.sh:11]: That works. Then I'll do a separate PR to remove the qualification.

**Review by mschrimpf — APPROVED** [review]
Approving assuming that you will complete the TODO in the PR: `pull the latest lookup.csv to make sure we're not leaving any PRs behind` (e.g. https://github.com/brain-score/brainio_collection/pull/46 was just merged)

**Pull request merged**